src/grid/grid_replay.c

```diff
@@ -79,8 +85,10 @@ static double parse_double(const char key[], FILE *fp) {
 static void parse_double3(const char key[], FILE *fp, double vec[3]) {
   char line[100], format[100];
   read_next_line(line, sizeof(line), fp);
-  snprintf(format, sizeof(format), "%s %%le %%le %%le", key);
-  assert(sscanf(line, format, &vec[0], &vec[1], &vec[2]) == 3);
+  snprintf(format, sizeof(format), "%s %%f %%f %%f", key);
```

Comment:
While admittedly `scanf` also accepts scientific notation for `%f`, we do need the `l` modifier because we're reading doubles.

Reply:
Unfortunately, we do not seem to have `macOS on Apple M1 (psmp)` in our list of tests (like Dashboard only).

I am on a x86-64 macOS, and the issue seems to be not present there (beside of the OS/SDK/CLI not bleeding edge anymore aka regression). I have to find an M1/M2/M3 mac...